path_candidates_param:
    racingline_path: 'path_racingline'
    overtaking_candidate_path_array: ['path_a','path_c'] # do not include the racing line here
    overtaking_candidate_path_alias_array: ['alias_a','alias_c'] # ex) left-center / center / right 
    # left --> right order

planning_params:
    splining_interval: 1.0 # [m]
    follow_enable_dist: 50 # [m]
    planning_horizon_t: 4.0 # [sec]
    planning_dt: 0.25 # [sec]
    max_accel: 2.0 # [mpss]
    overtaking_longitudinal_margin: 15.0 # [m]
    overtaking_lateral_margin: 2.0 # [m]
    merging_longitudinal_margin: 15.0 # [m]
    merging_allow_dist_to_racingline: 1.0 # [m] : maximum euclidean distance allowes merging back to the raceline

collision_checking_params:
    overlap_checking_dist_bound: 5.0 # [m] 
    overlap_checking_time_bound: 1.0 # [sec]